test/wordGenerator.spec.js

```diff
@@ -0,0 +1,10 @@
+import {WordGenerator} from '../src/wordGenerator'
+import {WordsSpy} from '../test/wordsSpy'
+
+test("gives a random word", () => {
+  let words = new WordsSpy()
```

Comment:
This looks like more like Stub than a Spy

https://8thlight.com/blog/uncle-bob/2014/05/14/TheLittleMocker.html

Also, is it really random if you can know the result expected?

Maybe you could have something like:
`expect(wrodGenerator.randomWord()).toBeIncludedIn(["this word", "or may be this one"])`